pyro/contrib/timeseries/gp.py

```diff
@@ -111,3 +117,142 @@ def forecast(self, targets, dts):
         filtering_state = self._filter(targets)
         predicted_mean, predicted_covar = self._forecast(dts, filtering_state)
         return torch.distributions.Normal(predicted_mean, predicted_covar.sqrt())
+
+
+class LinearlyCoupledMaternGP(TimeSeriesModel):
+    """
+    A time series model in which each output dimension is modeled as a linear combination
+    of shared univariate Gaussian Processes with Matern kernels.
+
+    In more detail, the generative process is:
+
+        :math:`y_i(t) = \\sum_j A_{ij} f_j(t) + \\epsilon_i`
```

Comment:
Could you mention that `f_j` is typically lower dimensional than `y_i`? I think that could help understand the motivation.

Reply:
i mention this in the `num_gps` docstring